server/api/orders.spec.js

```diff
@@ -0,0 +1,96 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../db');
+const app = require('../index');
+const Order = db.model('order');
+
+describe('Order routes', () => {
+  beforeEach(() => {
+    return db.sync();
+  });
+
+  describe('/api/orders/', () => {
+    const name = 'Test Order';
+    const description = 'Test Description';
+    const price = 6.89;
+
+    beforeEach(() => {
+      return Product.create({
```

Comment:
Shouldn't this be Order?
Did it pass the tests?

Reply:
That's right,  I used Product sort of as a place holder because actually the order model wasn't finished. My thinking was we'd have to use fields from the Product model because wouldn't an order contain data which exists in Product?

Reply:
You still need to make an order instance though for the Tests to pass